sdcm/utils/user_profile.py

```diff
@@ -130,7 +130,12 @@ def get_profile_content(stress_cmd):
     else:
         raise FileNotFoundError(f"Profile is not found in stress command: '{stress_cmd}'")
 
-    cs_profile = find_file_under_sct_dir(filename=profile_file, sub_folder=profile_path)
+    if profile_path.startswith("/tmp"):
```

Comment:
there is some logic referring to /tmp also in `find_file_under_sct_dir()`

what is the difference between the two ?

Reply:
That logic there doesn't get reached because the provided file with custom dynamic suffix is tried to be found under the "sct" dir, obviously fails and proceeds with the final line of the func - raising exception.
That func works ok only if user profile file doesn't have dynamic parts in the name.